plugins/destination/gcs/client/spec/schema_test.go

```diff
@@ -143,6 +162,30 @@ func TestSpecJSONSchema(t *testing.T) {
 			Spec: `{"format": "csv", "path": "abc", "bucket": "abc", "batch_timeout":null}`,
 		},
 
+		// no_rotate + path({{UUID}})
+		{
+			Name: "no_rotate:false & path:{{UUID}}",
+			Spec: `{"format": "csv", "path": "{{UUID}}", "bucket": "b", "no_rotate":false}`,
+		},
+		{
+			Name: "no_rotate:true & path:{{UUID}}",
+			Spec: `{"format": "csv", "path": "{{UUID}}", "bucket": "b", "no_rotate":true}`,
+			Err:  true,
+		},
+		{
+			Name: "no_rotate:false & path:abc",
+			Spec: `{"format": "csv", "path": "abc", "bucket": "b", "no_rotate":false}`,
+		},
+		{
+			Name: "no_rotate:true & path:abc",
+			Spec: `{"format": "csv", "path": "abc", "bucket": "b", "no_rotate":true}`,
+		},
+		{
+			Name: "no_rotate:false & path:{{TABLE}}",
+			Spec: `{"format": "csv", "path": "{{TABLE}}", "bucket": "b", "no_rotate":false}`,
+			Err:  true,
```

Comment:
What's the reason this is returning an error and `no_rotate:false & path:abc` doesn't?
Is it to allow to default to the old behavior when there are no path variables and enforce `{{UUID}}` when there are?

Reply:
OK I think https://github.com/cloudquery/cloudquery/pull/17602/files#r1561583882 answers it